src/ResourceManagement/Insights/Insights/Microsoft.Azure.Insights.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Insights
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Insights">
-      <PackageVersion>0.11.0-preview</PackageVersion>
+      <PackageVersion>0.13.1-preview</PackageVersion>
```

Comment:
Version 0.12 was already released internally. Since my changes introduce api changes, I increased it.